### PR TITLE
Sept Updates to CalLogs

### PR DESCRIPTION
**Pull request by Shanefe**
Add Run Information about when and who ran the script.
Add links to Public Articles on CalLogs.
Updated Attendee Count function.
Worked on TimeLine for Exceptions.
Forced Restart of the script when they first install ImportExcel.

### CODE REVIEW

**Comment by v-rusil** [comment]
Commenting

File: Calendar/CalLogHelpers/CalLogCSVFunctions.ps1 [hunk 1]
```diff
@@ -223,3 +219,35 @@ function ConvertDateTime {
...
+    if ($From.SmtpAddress -ne "NotFound") {
+        return ($AttendeeCollection -split ';').Count
+    } else {
+        return "-"
+    }
+}
+
+function CompressedVersion {
+    [string] $CompressedString = ""
+    if ($CalLog.AppointmentSequenceNumber -eq "NotFound" -or [string]::IsNullOrEmpty($CalLog.AppointmentSequenceNumber)) {
```
Comment (dpaulson45) [inline Calendar/CalLogHelpers/CalLogCSVFunctions.ps1:236]: `$CalLog` is a local variable the function `BuildCSV` and `CompressedVersion` is not within that function. Therefore, we should pass the variable or move the function inside the `BuildCSV`

**Review by dpaulson45 — CHANGES_REQUESTED** [review]

File: Calendar/CalLogHelpers/ExportToExcelFunctions.ps1 [hunk 2]
```diff
@@ -88,6 +88,8 @@ function GetExcelParams($path, $tabName) {
         FreezeTopRow            = $true
         #  BoldTopRow              = $true
         Verbose                 = $false
+        Password                = "TEST"
+#        Password                = $script:Password
```
Comment (dpaulson45) [inline Calendar/CalLogHelpers/ExportToExcelFunctions.ps1]: Should this be here?

Comment (dpaulson45) [reply Calendar/CalLogHelpers/ExportToExcelFunctions.ps1]: NVM, removed

**Review by dpaulson45 — APPROVED** [review]

File: Calendar/Get-CalendarDiagnosticObjectsSummary.ps1 [hunk 3]
```diff
@@ -190,9 +195,15 @@ if (-not ([string]::IsNullOrEmpty($Subject)) ) {
     Write-Warning "A valid MeetingID was not found, nor Subject. Please confirm the MeetingID or Subject and try again."
 }
 
+if($ExportToExcel.IsPresent) {
+    Close-ExcelPackage $excel -Password "Test"
```
Comment (Shanefe) [inline Calendar/Get-CalendarDiagnosticObjectsSummary.ps1]: >ose-ExcelPackage $excel -Password "Test" [](http://example.com/codeflow?start=6&length=40)

This should not be here...

**Pull request merged**